### DISCUSSION TITLE
[Feature Request] Prefixing Messages With One Config Edit

### DISCUSSION THREAD

**Discussion by Rage-GitHub** [Zrips/CMI#3203] [Feature Requests]
What I mean by this is to have a config option maybe somewhere in the **config.yml** or **Locale_.yml** where you can change an option like **prefixchatmessage: true/false** and it will add the desired prefix to all chat messages. This would save a lot of time for edits to the Locale files and also allow a universal use for the Locale files so each one doesn't have to have a prefix set to it.

**Comment by Zrips** [comment]
Not really possible as not all messages should have a prefix and not all messages go into the chatbox. Like basic command help page doesn't need prefix, chat text editor doesn't need prefix and on and on and on and on. Unless you have some idea how we could differentiate between those lines and avoid printing prefix where we don't need.
